src/parsers/power_system_table_data.jl

```diff
@@ -969,6 +998,10 @@ function make_thermal_generator(data::PowerSystemTableData, gen, cost_colnames,
         time_limits = timelimits,
         operation_cost = op_cost,
         base_power = base_power,
+        ext = Dict(
```

Comment:
This will always add outage info to `ext`. I think we want to only add it if it exists and the columns are specified in the configuration.yaml file. So, I'd suggest making another function like `add_outage_info!(component, gen)` that adds the outage info to the component after it's created if the `(gen.fotr, gen.mttr) != (nothing,nothing)`

Reply:
Yeahm this makes sense. The new commit includes these changes.